src/API/OnboardingTasks.php

```diff
@@ -341,14 +341,28 @@ private static function sideload_homepage_images( $post_id, $number_of_images )
 
 		if ( ! empty( $profile['industry'] ) ) {
 			foreach ( $profile['industry'] as $selected_industry ) {
-				$images_to_sideload[] = ! empty( $available_images[ $selected_industry ] ) ? $available_images[ $selected_industry ] : $available_images['other'];
+				if ( ! is_array( $selected_industry ) || empty( $selected_industry['slug'] ) ) {
+					continue;
```

Comment:
Should we offer backwards compatibility here for users that have already completed the profiler and still have the task list enabled?  Or do we think this is more of an edge case?

Reply:
While I do think that's an edge case - it's not too hard to handle.

Reply:
Done in 7d3fbf0.